yuuki/proxy.py

```diff
@@ -43,31 +44,68 @@ def recieve():
     """
     if not request.json:
         abort(400)
-    
+
     response = PROFILE.dispatch(request.get_json())
     return jsonify(response), 200
 
+def parse_config(config_file):
+
+    """
+    Receive a file path and parse it to a dict
+    """
+
+    config_reader = SafeConfigParser()
+    config_reader.read(config_file)
+    config_dict = {}
+
+    for section_name in config_reader.sections():
+
+        config_dict[section_name] = {}
+
+        for name, value in config_reader.items(section_name):
+
+            config_dict[section_name][name] = value
+    
+    return config_dict
 
 def main():
+
     """
     Parse configuration and start flask app.
-
-    WARNING: only a single profile file is supported at this time
```

Comment:
Multiple profiles are currently subtly broken in dispatch.py; spreading the logic for *different* actions across multiple .py files should work correctly, but spreading the logic for the same action across multiple files will not; only the target/actuator types from the last profile will be work.

I'll create an issue for this and try to fix it later. I'd like to support multiple profiles, but in case people don't see the issues list, I want to keep a warning in the main() function docstring.